building/devices/zynqmp.rst

```diff
@@ -0,0 +1,115 @@
+.. _zynqmp_zcu102:
+
+#######################
+QEMU for ZynqMP zcu102
+#######################
+The instructions below show how to run OP-TEE using QEMU for ZynqMP zcu102
+board. 
+
+Setting up the toolchain
+************************
+This build chain heavily relies on Petalinux 2018.2 therefore the first step 
+is to download and install the Petalinux 2018.2 toolchain from the Xilinx
+website.
+By default, the makefile considers that Petalinux is installed in 
+``/opt/Xilinx/petalinux_2018_2``.
+Then, you have to download the zcu102 BSP file from the Xilinx website.
+
+You may have to create a free Xilinx account to proceed with the two
+previous steps.
```

Comment:
An URL telling where to create a Xilinx account?

Reply:
when you try to download petalinux and/or bsp xilinx website propose to either sign in or create an account